src/areaSearch/areaSearchSpecsPage.tsx

```diff
@@ -156,13 +178,15 @@ const AreaSearchSpecsPage = ({
             return <BlockLoader />;
           }
 
-          const onSubmit = (e: FormEvent): void => {
+          const onSubmit = async (e: FormEvent): Promise<void> => {
```

Comment:
Hmm, is this necessary? Considering that this doesn't return anything, I'm not sure what's the effective difference between sync and async here.